cmake/SetupBlaze.cmake

```diff
@@ -19,24 +19,32 @@ set_property(TARGET Blaze PROPERTY
   INTERFACE_INCLUDE_DIRECTORIES ${BLAZE_INCLUDE_DIR})
 set_property(TARGET Blaze PROPERTY
   INTERFACE_LINK_LIBRARIES Lapack)
+target_link_libraries(
+  Blaze
+  INTERFACE
+  Blas
+  GSL::gsl # for BLAS header
+  Lapack
+  )
 
 target_compile_definitions(Blaze
   INTERFACE
-  # Override SMP configurations
+  # Enable external BLAS kernels
+  BLAZE_BLAS_MODE=1
+  # Set default matrix storage order to column-major
+  BLAZE_DEFAULT_STORAGE_ORDER=blaze::columnMajor
+  # Disable SMP parallelization
   BLAZE_USE_SHARED_MEMORY_PARALLELIZATION=0
-  BLAZE_OPENMP_PARALLEL_MODE=0
-  BLAZE_CPP_THREADS_PARALLEL_MODE=0
-  BLAZE_BOOST_THREADS_PARALLEL_MODE=0
   # Disable MPI parallelization
   BLAZE_MPI_PARALLEL_MODE=0
-  # Disable HPX parallelization
-  BLAZE_HPX_PARALLEL_MODE=0
-  BLAZE_USE_PADDING=0
-  # Enable non-temporal stores for cache optimization of large data structures
-  BLAZE_USE_STREAMING=1
-  BLAZE_USE_OPTIMIZED_KERNELS=1
   # Skip initializing default-constructed structures for fundamental types
   BLAZE_USE_DEFAULT_INITIALIZATON=0
+  # Use BLAS header from GSL. We could also find and include a <cblas.h> (or
+  # similarly named) header that may be distributed with the BLAS
+  # implementation, but it's not guaranteed to be available and may conflict
+  # with the GSL header. Since we use GSL anyway, it's easier to use their BLAS
+  # header.
+  BLAZE_BLAS_INCLUDE_FILE=<gsl/gsl_cblas.h>
```

Comment:
what are the performance implications of using the cblas header? Does this mean Blaze will use CBlas instead of MKL or OpenBLAS?

Reply:
See https://github.com/sxs-collaboration/spectre/pull/3806#discussion_r808804170